extras/examples/linux/README.md

```diff
@@ -546,10 +744,10 @@ Alternatively you can use JSON pointer to access the same value:
     temperature: "/my_temperature"
 
 All keys are considered as JSON pointer when they start with "/", otherwise
-keys are expected to be name of parameter in the root structure.
+keys are expected to be named of parameter in the root structure.
```

Comment:
OK, przywracam, ale zasugerował mi to słownik ;)
`The past participle is required after "to be". `
`Incorrect:  `
`It can be use by everyone. `
`Correct:  `
`It can be used by everyone. `

Reply:
Ten przykład ze słownika dotyczył czasownika po "to be". Tutaj name (a name) to rzeczownik, ale bez "a" wyglądał bardziej na czasownik ;P